### DISCUSSION TITLE
Welcome to web-resources-project Discussions!

### DISCUSSION THREAD

**Discussion by jayk-gupta** [jayk-gupta/web-resources-project#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dneth04** [comment]
Hi @jayk-gupta , I am Nethradevi a 2nd year BTech CS undergrad at BITS Pilani. I am into Web dev and I chose this community to contribute my knowledge and help with the enhancement of the repository. I really like the initiative of this website which will surely be a most required website for any student out there and I have completely gone through the README file and contributing guidelines. I have also already implemented them in my system. I am now just looking forward to being assigned to the [ISSUE](https://github.com/jayk-gupta/web-resources-project/issues/190#issue-1724594765)  I raised a few minutes ago and I am very excited to start contributing to this community!

I can help the community with Bugs solving ,Styling tools, documentation, Adding new features and other tasks if required. I am a Focused Student and I am always willing to learn new things. I am confident that I can make a valuable contribution to this community.

Really excited to be part of GSSOC'23 , and hopefully I can contribute the best to this project :)
